packages/core/template/portfolio/index.md

```diff
@@ -0,0 +1,43 @@
+<frontmatter>
+  title: Home Page
```

Comment:
Keep in mind the use of frontmatter :https://markbind.org/userGuide/tweakingThePageStructure.html#frontmatter and omitFrontmatter 

(btw - if you're looking for more issues to pick up, you can consider adding to the discussion here: https://github.com/MarkBind/markbind/issues/2295 )

Anyway, because of the use of includes in this page, the page title is not Home Page but that of the last included file. Could you go through and remove irrelevant frontmatter? Might be worthwhile to also include in a code comment the purpose of frontmatter on this page